src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import TextEditor from './pages/index';
```

Comment:
did not use index for import - use (import TextEditor from './pages')
export all pages from index.js

Reply:
Done